src/components/layout/SuperChat.js

```diff
@@ -0,0 +1,181 @@
+import { React, useState, useRef, Fragment } from "react";
+
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Paper from "@material-ui/core/Paper";
+import { Typography } from "@material-ui/core";
+import Toolbar from "@material-ui/core/Toolbar";
+// import Button from '@material-ui/core/Button';
+import ForumRoundedIcon from "@material-ui/icons/ForumRounded";
+import InputBase from "@material-ui/core/InputBase";
+import IconButton from "@material-ui/core/IconButton";
+import SendIcon from "@material-ui/icons/Send";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  bar: {
+    backgroundColor: theme.palette.secondary.dark,
+  },
+  icon: {
+    marginRight: theme.spacing(0.5),
+  },
+  title: {
+    flexGrow: 1,
+    display: "flex",
+    justifyContent: "center",
+  },
+  t2: {
+    display: "flex",
+  },
+  chatview: {
```

Comment:
Mention styles in components only if dynamic values or style bindings are used. Make sure to have static styles to one or multiple css files